packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <nav v-if="userId" :aria-label="$gettext('Account menu')">
+    <oc-button
+      id="_userMenuButton"
+      ref="menuButton"
+      class="oc-topbar-personal"
+      appearance="raw"
+      :aria-label="$gettext('User Menu')"
+    >
+      <avatar-image
+        class="oc-topbar-personal-avatar oc-flex-inline oc-flex-center oc-flex-middle"
+        :width="32"
+        :userid="userId"
+        :user-name="user.displayname"
+      />
+    </oc-button>
+    <oc-drop
+      ref="menu"
+      drop-id="account-info-container"
+      toggle="#_userMenuButton"
+      mode="click"
+      close-on-click
+      :options="{ pos: 'bottom-right', delayHide: 0 }"
+      padding-size="small"
+    >
+      <oc-list class="user-menu-list">
+        <li>
+          <oc-button
+            id="oc-topbar-account-manage"
+            type="router-link"
+            :to="{ path: '/account' }"
+            appearance="raw"
+          >
+            <avatar-image :width="32" :userid="userId" :user-name="user.displayname" />
+            <span class="profile-info-wrapper" :class="{ 'oc-py-xs': !user.email }">
+              <span v-text="user.displayname" />
+              <br v-if="user.email" />
+              <span v-if="user.email" class="oc-text-small" v-text="user.email" />
+            </span>
+          </oc-button>
+        </li>
+        <li v-for="(n, nid) in applicationsList" :key="`user-menu-${nid}`">
```

Comment:
not a fan of using shorthands like `n` and `nid`. I would prefer to have something human readable here

Reply:
Changed to `app, index`